src/patternfly/components/FormControl/form-control.scss

```diff
@@ -185,6 +222,15 @@ $pf-c-form-control--m-search--Coordinates: "M505 442.7L405.3 343c-4.5-4.5-10.6-7
     background-position: var(--pf-c-form-control--invalid--BackgroundPosition);
     background-size: var(--pf-c-form-control--invalid--BackgroundSize);
     border-bottom-width: var(--pf-c-form-control--invalid--BorderBottomWidth);
+
+    &.pf-m-icon {
+      --pf-c-form-control--PaddingRight: var(--pf-c-form-control--m-icon--icon--PaddingRight);
+
+      background-image: var(--pf-c-form-control--invalid--BackgroundUrl), var(--pf-c-form-control--m-icon--BackgroundUrl);
+      background-repeat: no-repeat;
+      background-position: var(--pf-c-form-control--invalid--BackgroundPosition), calc(var(--pf-c-form-control--m-icon--BackgroundPositionX) - var(--pf-c-form-control--m-icon--icon--spacer) - var(--pf-c-form-control--invalid--BackgroundSizeX)) var(--pf-c-form-control--m-icon--BackgroundPositionY);
```

Comment:
should the calc here be as a variable for `background-position-x`

Reply:
Yep, good call!